### PR TITLE
#869: Display Translation Form on All Languages Description:

### PR DESCRIPTION
**Pull request by MiguelAxcar**
Following the enhancements made in ticket #322, this pull request extends the translation prompt functionality to all posts on the site, regardless of language. 

### Changes
The pull request streamlines the code and ensures that the "Translate This Post" prompt is displayed across all languages, including any new languages added to the site:
- Code Refactoring: The original code was simplified and refactored to directly retrieve languages without translations, making the process more efficient.
- Universal Translation Form: Previously, the translation prompt was limited to the English version of the site. The changes ensure that if a post is untranslated in any language, the translation form is displayed.

### Testing instructions
Verify if
- The translation prompt appears on all posts across languages.
- The feature seamlessly supports new languages as they are added to the site.

### CODE REVIEW

File: template-parts/content.php [hunk 1]
```diff
@@ -7,36 +7,21 @@
...
-	// Remove languages that have translations.
-	foreach ( $raw_languages as $sub_key => $sub_array ) {
-		if ( false === $sub_array['no_translation'] ) {
-			unset( $raw_languages[ $sub_key ] );
+	// Retrieve the complete list of languages in raw format, including their translation status.
+	$all_languages = pll_the_languages( array( 'raw' => 1 ) );
+	// Initialize a new associative array to store languages which translation are miss.
+	$languages_without_translations = array();
+
+	// Loop througout each language in the list.
```
Comment (goldenapples) [inline template-parts/content.php:17]:
```suggestion
	// Loop throughout each language in the list.
```

File: template-parts/content.php [hunk 2]
```diff
@@ -7,36 +7,21 @@
...
+	foreach ( $all_languages as $language ) {
+		// If the 'no_translation' flag is set true for a specific language, this language miss translation.
+		if ( $language['no_translation'] ) {
+			// Add the untranslated language to the associactive vector, mapping slug>name.
```
Comment (goldenapples) [inline template-parts/content.php:21]:
```suggestion
			// Add the untranslated language to the associative vector, mapping slug => name.
```

**Review by goldenapples — APPROVED** [review]
Works as I'd expect, and the code is cleaner and easier to read. i just called out a couple typos in code comments but nothing that blocks merging.

**Review by kadamwhite — APPROVED** [review]
Approving the code that isn't mine, but could use a final 👁️ from somebody who's not me to validate my own additions :)

**Review by goldenapples — APPROVED** [review]
I guess the only changes that actually have an impact are removing the `pll_default_language() === pll_current_language()` from the conditionals before showing the link or form for translation. But the simpler method of building the languages array looks nice too.

**Comment by kadamwhite** [comment]
@goldenapples The removal of the conditional was the big change today, we didn't properly capture @ckoerner's request to make sure that displayed cross-language when we last worked on this. Thanks for the review.

**Pull request merged**